### DISCUSSION TITLE
age relabelling OM simulations

### DISCUSSION THREAD

**Discussion by vbartolino** [NOAA-EDAB/ms-keyrun#91] [Ideas]
As described in #59 "... Atlantis is calculating true age for each species, not a Jan 1 birthday age".

![scheme](https://github.com/NOAA-EDAB/ms-keyrun/assets/9845667/c2287171-5ccf-4ee4-8060-c79ebe2ab51f)

I think it is relevant to align to the 1 Jan birthday convention and keep a calendar year in the model. Surveys are run in the month 3 and 8. This is the age relabelling that I'm considering for the survey data on cod, haddock and capelin before their use in the estimation model:

COD: spawn month 1 recruit month 6 age1 --> age0 in the fall survey
HAD: spawn month 3 recruit month 6 age1 --> age0 in the fall survey
CAP: spawn month 2 recruit month 14 age1 --> stay age1 in both spring and fall survey

**Comment by sgaichas** [comment]
This looks reasonable to me. I am testing a length structured model so I get around the issue that way. :-) 
If this sounds like a good convention for the other age structured models I think we can mark it answered and have everyone use the same convention. It might be worth making a table for all 11 species if anyone gets to a larger model, but not needed right now.
